src/app/starring/starring.component.ts

```diff
@@ -117,8 +119,14 @@ export class StarringComponent implements OnInit, OnDestroy, OnChanges {
       this.setStar().subscribe(
```

Comment:
`this.alertService.start('Starring/unstarring tool/workflow')` right before the subscribe.  Otherwise the snackbar errors I see right now are: 
![image](https://user-images.githubusercontent.com/24548904/51495320-e4ed6900-1d89-11e9-9cd8-245d433e51f5.png)

and if clicked again:
![image](https://user-images.githubusercontent.com/24548904/51495155-50830680-1d89-11e9-9da9-f2c6775cc26a.png)

The alert service has a short description on how to use it

Reply:
I put the alertService.start in the file where setStar is defined in the new commit.I can put it in the starring.component.ts instead if you think it is an issue.

Reply:
Nevermind, moved it back to starring.component.ts. Realized having it where setStar is defined was causing tests in travis to fail. Not sure why, but should be working now.